desk/app/groups-ui.hoon

```diff
@@ -227,6 +237,12 @@
     =.  hidden-contact-suggestions
       (~(put in hidden-contact-suggestions) ship)
     cor
+  ::
+      %ui-add-contact-suggestions
+    =+  ship-list=!<((list @p) vase)
+    =.  manual-contact-suggestions
+      (~(uni in manual-contact-suggestions) (sy ship-list))
```

Comment:
`+gas:in` exists for adding a list of entries into a set. Should be completely equivalent, except more idiomatic and maybe very slightly more performant.

```suggestion
    =.  manual-contact-suggestions
      (~(gas in manual-contact-suggestions) ship-list)
```

Reply:
Ahh, good one to know!